src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageFormView.tsx

```diff
@@ -182,18 +216,21 @@ const CreateWorkPackageFormView: React.FC<CreateWorkPackageFormViewProps> = ({ a
               />
             </FormControl>
           </Grid>
-          <Grid item xs={12} md={6}>
+          <Grid item xs={12} md={4}>
             <FormControl fullWidth>
               <FormLabel>Project WBS Number</FormLabel>
-              <ReactHookTextField
-                name="wbsNum"
-                control={control}
-                placeholder="Enter project WBS number..."
-                errorMessage={errors.wbsNum}
+              <NERAutocomplete
+                id="wbs-autocomplete"
+                onChange={wbsAutocompleteOnChange}
+                options={wbsDropdownOptions}
+                size="small"
+                placeholder="Select a project or work package"
+                value={wbsDropdownOptions.find((element) => element.id === wbsNum) || null}
+                sx={{ width: 1 / 2 }}
```

Comment:
<img width="937" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/0ce06e6c-8d1c-422e-a4f4-b77c02204183">
this looks goofy rn, the autocomplete should take up more space so you can actual see the project title